web-ui/src/pages/CheckinsReportPage.jsx

```diff
@@ -24,8 +26,34 @@ const CheckinsReportPage = () => {
     const bPieces = b.name.split(' ').slice(-1);
     return aPieces.toString().localeCompare(bPieces);
   });
+
   const [filteredPdls, setFilteredPdls] = useState(pdls);
 
+  const processedQPs = useRef(false);
+  useQueryParameters(
```

Comment:
This adds support for the `pdl` query parameter which keeps track of the PDLs the user has selected.